bigquery/api/SimpleApp/runTests.ps1

```diff
@@ -11,12 +11,6 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations under
 # the License.
-Import-Module ..\..\..\BuildTools.psm1 -DisableNameChecking
 
-
-BackupAndEdit-TextFile @(".\Program.cs") `
-    @{"YOUR-PROJECT-ID" = $env:GOOGLE_PROJECT_ID} `
-{
-    dotnet restore --force
-    dotnet test --no-restore --test-adapter-path:. --logger:junit 2>&1 | %{ "$_" }
-}
+dotnet restore --force
+dotnet test --no-restore --test-adapter-path:. --logger:junit 2>&1 | %{ "$_" }
```

Comment:
nit: We have removed the newline from the end of the file.

Reply:
Done